packages/opencensus-instrumentation-https/test/test-https.ts

```diff
@@ -42,7 +43,10 @@ function doNock(
 const httpRequest = {
   get: (options: {}|string) => {
     return new Promise((resolve, reject) => {
-      return https.get(options, resp => {
+      if (typeof (options) === 'string') {
```

Comment:
We should test both `request` and `get`, rather than just one. In Node 6, you may get unexpected results: https://github.com/nodejs/node/blob/master/lib/https.js#L288 `get` may not be patched if only `request` is patched.

Reply:
Yes, you are right. I made this change and added tests for both `get()` and `request()` methods. Thanks for pointing it out.